WcaOnRails/.rubocop_todo.yml

```diff
@@ -234,6 +234,7 @@ Style/ClassAndModuleChildren:
     - 'app/controllers/accounts/registrations_controller.rb'
     - 'app/controllers/api/v0/api_controller.rb'
     - 'app/controllers/api/v0/competitions_controller.rb'
+    - 'app/controllers/api/v0/records_controller.rb'
```

Comment:
It looks quite better indeed, but some problems with constant scope can emerge. See [this article](http://blog.honeybadger.io/avoid-these-traps-when-nesting-ruby-modules/).